spec/facter/resolvers/processors_spec.rb

```diff
@@ -6,18 +6,31 @@
   end
 
   context 'when on x86 architecture' do
+    let(:log_spy) { instance_spy(Facter::Log) }
     let(:processors) { 4 }
     let(:models) do
       ['Intel(R) Xeon(R) CPU E5-2697 v4 @ 2.30GHz', 'Intel(R) Xeon(R) CPU E5-2697 v4 @ 2.30GHz',
        'Intel(R) Xeon(R) CPU E5-2697 v4 @ 2.30GHz', 'Intel(R) Xeon(R) CPU E5-2697 v4 @ 2.30GHz']
     end
     let(:physical_processors) { 1 }
+    let(:cores_per_socket) { 1 }
+    let(:threads_per_core) { 2 }
+    let(:lscpu_output) do
+      ["'Thread(s) per core': 2",
+       "'Cores(s) per socket': 1"].join("\n")
+    end
```

Comment:
Sorry, just saw these tests. Since you're testing a different resolver here (`ProcessorsLscpu`) it would be better to have the tests in a separate file.
Also, should we change the name of the resolver to just `lscpu`, since that's what the resolver executes?